src/scripts/utils/listeners.js

```diff
@@ -0,0 +1,19 @@
+import { involvement } from '../API/Involvement.js';
+
+class Listeners {
+  addLikeListener = (event) => {
+    event.addEventListener('click', () => {
+      involvement.setLike(event.dataset.id);
+      const likesToUpdate = document.querySelector(`.movie-${event.dataset.id}`);
+      involvement.getLikesOf(event.dataset.id)
+        .then((likes) => {
+          likesToUpdate.innerHTML = `${likes} people like this`;
+        });
+    });
+  }
+}
+
+const elisteners = new Listeners();
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Please remove this eslint disable line its best to fix the issue in the code.